src/bin/service.rs

```diff
@@ -58,7 +51,7 @@ fn main() {
     let config = settings
         .build_rocket_config()
         .expect("Error creating rocket config");
-    rocket::custom(config)
+    rocket::custom(config, settings.log.level.0.as_str() != "off")
```

Comment:
Is the `.0` here going to conflict with you other PR where you remove the use of this `.0` style?